### PR TITLE
Support combinations of short flags for auto-completion providers

### PR DESCRIPTION
**Pull request by mkenne11**
Added function 'combineShortFlags' to support entry of short flag combinations for auto-complete providers.

### CODE REVIEW

**Comment by mkenne11** [comment]
This is a possible implementation to address #596.

File: src/plugins/autocompletion_utils/Common.ts [hunk 1]
```diff
@@ -386,3 +386,24 @@ export const commandWithSubcommands = (subCommands: SubcommandConfig[]) => {
...
+    return async (context: AutocompletionContext) => {
+        const token = context.argument.value;
+        const reShortFlag = new RegExp(/^\-[a-zA-Z]$/);
+        const reShortFlags = new RegExp(/^\-[a-zA-Z]+$/);
+        const suggestions = await suggestionsProvider(context);
+        if (reShortFlags.test(token)) {
+            return suggestions
+                    .filter(s =>
+                        reShortFlag.test(s.value) && !token.includes(s.value.slice(1))
+                            && s.withSpace)
```
Comment (mkenne11) [inline src/plugins/autocompletion_utils/Common.ts]: I am assuming the (suggestion) `withSpace` attribute is used for short flag options with values, so these should not be included when suggesting combined short flag auto-complete options.

Comment (drew-gross) [inline src/plugins/autocompletion_utils/Common.ts]: That is correct

**Comment by drew-gross** [comment]
This looks awesome! Thanks for the help!

**Pull request merged**